server/wfprev-api/src/main/java/ca/bc/gov/nrs/wfprev/SecurityConfig.java

```diff
@@ -39,12 +43,15 @@ public class SecurityConfig {
     @Value("${security.oauth.authTokenUrl}")
     private String authTokenUrl;
 
+    @Value("${spring.application.baseUrl}")
+    private String baseUrl;
+
     @Bean
     AuthenticationEntryPoint authenticationEntryPoint() {
         BasicAuthenticationEntryPoint result;
 
         result = new BasicAuthenticationEntryPoint();
-        result.setRealmName("wfim-incidents-api");
+        result.setRealmName("wfprev-api");
```

Comment:
These changes and the security class is already in another PR. Would be better to merge that in first before modifying

Reply:
Reverted for now